templates/wordpress/files/composer.json

```diff
@@ -0,0 +1,22 @@
+{
```

Comment:
This file comes from upstream so we shouldn't be overwriting it.  Modifying via script only.

Reply:
I don't understand why we do this. Using one composer file to pull another dependency with a composer file is design intent of Composer. In this case, we set the relevant install paths for the upstream in our file. This is safer than overwriting the source, IMHO.